walk/config.go

```diff
@@ -104,6 +112,46 @@ func (cr *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 	c.Exts[walkName] = wcCopy
 }
 
+func (c *Configurer) loadBazelIgnore(repoRoot, rel string, wc *walkConfig) error {
+	// Only load .bazelignore at the root, and only once
+	if rel != "" {
+		return nil
+	}
+
+	ignorePath := path.Join(repoRoot, ".bazelignore")
+	file, err := os.Open(ignorePath)
+	if os.IsNotExist(err) {
+		return nil
+	}
+	if err != nil {
+		return fmt.Errorf(".bazelignore exists but couldn't be read: %v", err)
+	}
+	defer file.Close()
+
+	wc.excludes = append(wc.excludes, path.Join(rel, ".bazelignore"))
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		ignore := strings.TrimSpace(scanner.Text())
+		if ignore == "" || string(ignore[0]) == "#" {
+			continue
+		}
+		// Bazel ignore paths are always relative to repo root.
+		// Glob patterns are not supported.
+		pat := path.Join(rel, ignore)
```

Comment:
I could be wrong here but I think .bazelignore pattern is counting from repoRoot and not from rel?

https://docs.bazel.build/versions/main/guide.html#bazelignore

Also if we have established `rel` to be effectively empty in begining of the func, do we still need this?

Reply:
Agreed, this logic actually only works when `rel` is empty so we can probably just use the `ignore` value directly. I think I copied this from the original